resources/elasticsearch/serverless.yml

```diff
@@ -65,7 +82,7 @@ resources:
       DeletionPolicy: Retain
       Properties:
         DomainName: ${self:custom.logsDomain}
-        ElasticsearchVersion: 5.5
+        ElasticsearchVersion: 6.0
```

Comment:
I think this will break and will need '6.0' quotes, at least it broke for me previously.